data/data/manifests/openshift/cluster-networkconfig-crd.yaml

```diff
@@ -0,0 +1,19 @@
+# This is the advanced network configuration CRD
```

Comment:
This should not be created by installer right? only `network.config.openshift.io` 

Looks like we are dropping the `netopv1.NetworkConfig` from vendor as installer no longer creates that...

Reply:
I kept it here to make life easier for users who need to create advanced network settings; mind if we keep it for now and see how the docs shake out?